### PR TITLE
Added configuration instructions for Blackfire.io in Cloud Docker

### PR DESCRIPTION
**Pull request by meker12**
## Purpose of this pull request

- Added configuration instructions for Blackfire.io in Cloud Docker

## Affected DevDocs pages

- https://devdocs.magento.com/cloud/docker/docker-config-blackfire-io.html

## Links to Magento source code

https://github.com/magento/magento-cloud-docker/pull/202

whatsnew
Added basic instructions for configuring and using Blackfire.io for performance testing in a Magento Cloud Docker environment. See [Add Blackfire.io to Magento Cloud Docker](https://devdocs.magento.com/cloud/docker/docker-config-blackfire-io.html).

### CODE REVIEW

File: src/cloud/docker/docker-config-blackfire-io.md [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+group: cloud-guide
+title: Add Blackfire.io to Docker
+functional_areas:
+- Cloud
+- Setup
+- Configuration
+
+---
+
+You can add Blackfire.io to your {{site.data.var.mcd-prod}} environment to complete automated performance testing.
```
Comment (drpayyne) [inline src/cloud/docker/docker-config-blackfire-io.md:11]:
```suggestion
You can add Blackfire.io to your {{site.data.var.mcd-prod}} environment to get completely automated performance testing.
```

File: src/cloud/docker/docker-config-blackfire-io.md [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+
+[Blackfire.io for Magento Cloud](https://blackfire.io/magento) is a PHP profiler and automated performance testing tool that can be used in the development Integration, Staging, and Production environments. It enables you to locate and investigate performance issues in your environment at the code level and creates a performance profile by tracking every PHP call, method, and SQL query performed by your code. Blackfire digs deeper to provide granular performance analytics.
```
Comment (drpayyne) [inline src/cloud/docker/docker-config-blackfire-io.md:13]:
```suggestion
[Blackfire.io for Magento Cloud](https://blackfire.io/magento) is a PHP profiler and automated performance testing tool that can be used in the development, integration, staging, and production environments. It enables you to locate and investigate performance issues in your environment at the code level and creates a performance profile by tracking every PHP call, method, and SQL query performed by your code. Blackfire also digs deeper to provide granular performance analytics.
```

File: src/cloud/docker/docker-config-blackfire-io.md [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+1. Clear the cache.
+
+   ```bash
+   docker-compose run deploy magento-command cache:clean
+   ```
+
+1. Make sure necessary containers are up and running.
+
+   ```bash
+   docker ps
```
Comment (drpayyne) [inline src/cloud/docker/docker-config-blackfire-io.md:94]:
```suggestion
   docker-compose ps
```

**Comment by meker12** [comment]
> It looks like there is a lot more than Backfire in ∂ocker here.

Changed to Docker integration base branch.

File: src/cloud/docker/docker-config-blackfire-io.md (hunk 3 above)
Comment (drpayyne) [reply src/cloud/docker/docker-config-blackfire-io.md:94]: `docker-compose ps` shows all containers applicable to current scope including downed/stopped containers to get a better understanding of docker status.

**Comment by drpayyne** [comment]
Hey @meker12, I have suggested some changes. Please review them. Thanks!

File: src/cloud/docker/docker-config-blackfire-io.md (hunk 1 above)
Comment (hguthrie) [reply src/cloud/docker/docker-config-blackfire-io.md:11]: "get" is too ambiguous. If you are suggesting that this needs improvement, then I would understand this more clearly:
```suggestion
You can add Blackfire.io to your {{site.data.var.mcd-prod}} environment to fully automate performance testing.
```

**Pull request merged**